examples/cloud-armor/README.md

```diff
@@ -0,0 +1,18 @@
+# White listing ip's using Cloud Armor
+
+This is an example of setting up a project to take advatange of one of the [Cloud Armor features](https://cloud.google.com/armor/) that allows whitelisting of traffic to a compute instance based on ip address. It will set up a single compute instance running nginx that is accesible via a load balanced pool that is managed by cloud armor security policies.
+
+To run the example:
+* Set up a gcloud account with the compute engine api enabled
+* Update the the `variables.tf` or provide overrides in the command line 
+* Run with a command similar to:
+```
+terraform apply \
+	-var="region=us-west1" \
+	-var="region_zone=us-west1-a" \
+	-var="project_name=my-project-id-123" \
+```
+
+After running `terraform apply` the external ip address of the load balancer will be output to the console. Either enter the ip address into the broswer directly or add it to the hosts file on your machine so that it can be access at 'mysite.com'.
+
+Navigating to the address either way should result in a 403 rejection. Change the ip address in the whitelist rule in `main.tf` to a local ip address or range and re-run `terraform apply` to be able to hit the nginx welcome page on the instance. Note: it can take a little while before the changes apply.
```

Comment:
TBH I'm not completely sure. I noticed the policy gets updated quite quickly and correctly in the GCP UI, but it takes a handful of seconds before hitting the url will result in the correct action. I'm assuming this is because of some caching of the security rules, so I'm not sure if it's bug. Maybe I could try and see if there is documentation on how long this is expected to be?

Reply:
Ah, got it. That's fine then.